crates/bevy_winit/src/winit_windows.rs

```diff
@@ -25,6 +27,37 @@ impl WinitWindows {
         #[cfg(not(target_os = "windows"))]
         let mut winit_window_builder = winit::window::WindowBuilder::new();
 
+        let default_resize_constraints = WindowResizeConstraints::default();
```

Comment:
This should use the `WindowDescriptor`, right?

Reply:
This is just for some checks to force the minunum size and fix that crash at 0 height